modules/apps/portal-language/portal-language-override-web/src/main/resources/META-INF/resources/edit.jsp

```diff
@@ -0,0 +1,198 @@
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+EditDisplayContext editDisplayContext = (EditDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+portletDisplay.setShowBackIcon(Validator.isNotNull(editDisplayContext.getBackURL()));
+portletDisplay.setURLBack(editDisplayContext.getBackURL());
+
+renderResponse.setTitle(editDisplayContext.getPageTitle());
+%>
+
+<portlet:actionURL name="editPortalLanguageOverride" var="editURL" />
+
+<clay:container-fluid>
+	<liferay-frontend:edit-form
+		action="<%= editURL %>"
+		method="POST"
+		name="editPortalLanguageOverrideFm"
+	>
+		<aui:input name="redirect" type="hidden" value="<%= editDisplayContext.getBackURL() %>" />
+
+		<liferay-frontend:edit-form-body>
+			<clay:sheet-header>
+				<h2 class="sheet-title"><liferay-ui:message key="language-key-information" /></h2>
+			</clay:sheet-header>
+
+			<clay:sheet-section>
+				<clay:content-row
+					containerElement="h3"
+					cssClass="sheet-subtitle"
+				>
+					<clay:content-col
+						expand="<%= true %>"
+					>
+						<span class="heading-text"><liferay-ui:message key="key" /></span>
+					</clay:content-col>
+				</clay:content-row>
+
+				<clay:content-row
+					containerElement="div"
+					cssClass=""
+				>
+					<clay:content-col
+						expand="<%= true %>"
+					>
+						<c:choose>
+							<c:when test="<%= Validator.isNotNull(editDisplayContext.getKey()) %>">
+								<aui:input name="key" type="hidden" value="<%= editDisplayContext.getKey() %>" />
+
+								<span><%= editDisplayContext.getKey() %></span>
+							</c:when>
+							<c:otherwise>
+								<div class="form-group">
+									<aui:input name="key" pattern="[^ ]+" required="<%= true %>" value="<%= editDisplayContext.getKey() %>" wrapperCssClass="mb-0" />
+
+									<div class="form-feedback-group">
+										<div class="form-text">
+											<liferay-ui:message key="a-language-key-cannot-contain-any-whitespace" />
+										</div>
+									</div>
+								</div>
+							</c:otherwise>
+						</c:choose>
+					</clay:content-col>
+				</clay:content-row>
+			</clay:sheet-section>
+
+			<clay:sheet-section>
+				<clay:content-row
+					containerElement="h3"
+					cssClass="sheet-subtitle"
+				>
+					<clay:content-col
+						expand="<%= true %>"
+					>
+						<span class="heading-text"><liferay-ui:message key="translation-override" /></span>
+					</clay:content-col>
+
+					<c:if test="<%= Validator.isNotNull(editDisplayContext.getKey()) %>">
+						<clay:content-col>
+							<span class="heading-end">
+								<liferay-ui:icon
```

Comment:
We can remove the custom script at the end of this jsp by using the `liferay-ui:icon-delte` tag.

Reply:
Oh, I forgot about that. Thanks!